engine/src/main/java/org/terasology/rendering/dag/nodes/LateBlurNode.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.nodes;
+
+import org.terasology.assets.ResourceUrn;
+import org.terasology.config.Config;
+import org.terasology.config.RenderingConfig;
+import org.terasology.registry.In;
+import org.terasology.rendering.nui.properties.Range;
+import org.terasology.rendering.opengl.FBOConfig;
+import org.terasology.rendering.opengl.fbms.DisplayResolutionDependentFBOs;
+
+/**
+ * This class is a thin facade in front of the BlurNode class it inherits from.
+ * The term "late" is due to the fact that this type of nodes is used near the
+ * end of the rendering process leading to an image shown on the user display.
+ *
+ * Given an input FBO a blurred version of it will be stored in the given output FBO.
+ * Eventually the blurred version can be used for blur-based effects such as
+ * Depth of Field.
+ *
+ * For more information on Blur: https://en.wikipedia.org/wiki/Box_blur
+ * For more information on DoF: http://en.wikipedia.org/wiki/Depth_of_field
+
```

Comment:
Thanks again for detailed comments. 👍 A little ashamed to not be able to write those.